src/main/java/nbgame/game/Game.java

```diff
@@ -0,0 +1,181 @@
+package nbgame.game;
+
+import nbgame.constant.FileAccess;
+import nbgame.engine.*;
+import nbgame.gui.GameForm;
+import nbgame.gui.IntroForm;
+import nbgame.ship.Ship;
+import nbgame.user.BattleKeyboard;
+import nbgame.user.BattleMouse;
+import nbgame.user.Movable;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+public class Game {
+    private final GameForm gameForm;
+    private final ShipArrangement shipArrangement;
+    private final GraphicDriver graphicDriver;
+    private final Settings settings;
+    private final FileDriver fileDriver;
+
+    private final Movable userGameInterface = new BattleMouse(this);
+
+    private final BattleField gamerBattleField = new BattleField(this, FileAccess.FIELD_WAVE_PIC_PATH);
+    private final BattleField computerBattleField = new BattleField(this, FileAccess.FIELD_WAVE_PIC_PATH, userGameInterface);
+
+    private Round round;
+    private Status status;
+    private List<Ship> gamerShips;
+    private List<Ship> computerShips;
+    private List<Statistic> statistics;
+
+    public Game() {
+        settings = new Settings(2, 3, 4, 4, "None");
```

Comment:
co to jest 2, 3, 4, 4, "None"? Proponuję wprowadzić stale w tej klasie o odpowiednich nazwach. Unikamy tzw. "magic numbers" w kodzie